src/app/business/delfin/delfin.service.ts

```diff
@@ -435,5 +435,114 @@ export class DelfinService {
     metricsConfig(id, params): Observable<any>{
         return this.http.put(this.delfinStoragesUrl + '/' + id + '/metrics-config', params);
     }
+
+    //Get All Masking Views
+    
+    getAllMaskingViews(storageId, nativeMaskingViewId?, nativeStorageHostId?, nativeStorageHostGroupId?, nativeStorageVolumeGroupId?, nativeStoragePortGroupId?, name?, limit?, offset?, sort?): Observable<any> {
```

Comment:
Currently nativeStorageVolumeGroupId, nativeStoragePortGroupId filters not supported, hope it does not affect